src/ExtensionsSample/Program.cs

```diff
@@ -112,5 +116,39 @@ private static void EnsureSampleDirectoriesExist(string rootFilesPath)
 
             File.WriteAllText(Path.Combine(rootFilesPath, "input.txt"), "WebJobs SDK Extensions!");
         }
+
+        private static bool CheckForApiHub(JobHostConfiguration config)
```

Comment:
No this means if there is no valid connection string just use the local file system. and then everything just works fine.